### PR TITLE
Add tox for testing package install

### PR DESCRIPTION
**Pull request by kitchoi**
- Added tox settings
- Added MANIFEST.in
- Updated setup.py for package data
- Added __init__ for a few test packages
- Fix one local import for tests in py3

### CODE REVIEW

**Comment by kitchoi** [comment]
@sjagoe can you take a look please?  The tarball on pypi at the moment has missing files and fail tests.  tox can ensure that doesn't happen again.  thanks @cournape for his help!

File: setup.py [hunk 1]
```diff
@@ -153,7 +153,12 @@ def write_version_py(filename=_VERSION_FILENAME):
         install_requires = __requires__,
         license = "BSD",
         packages = find_packages(),
-        package_data = {'': ['images/*', '*.ini',]},
+        package_data = {'': ['images/*', '*.ini',],
```
Comment (sjagoe) [inline setup.py]: I think this is going to include `tox.ini` in eggs, whereas I expect we just want it in the sdist.

Comment (sjagoe) [inline setup.py]: Specifying it only in MANIFEST.in as `include tox.ini` may be a better approach.

Comment (sjagoe) [inline setup.py]: ah, I see I was wrong actually. It does not include top-level ini files. Ignore the above comment.

**Comment by sjagoe** [comment]
@kitchoi Please add a changelog entry

File: setup.py (hunk 1 above)
Comment (kitchoi) [inline setup.py]: I don't think tox.ini would be included because it is not inside the envisage package.

**Comment by kitchoi** [comment]
@sjagoe I have updated changelog and I checked the egg problem with setuptools bdist_egg.  Please let me know if I should do something different.

**Comment by sjagoe** [comment]
LGTM. I'll produce a 4.5.1 release.

**Pull request merged**